tgui/packages/tgui/routes.js

```diff
@@ -57,14 +56,7 @@ export const getRoutedComponent = store => {
   while (!esModule && interfacePathBuilders.length > 0) {
     const interfacePathBuilder = interfacePathBuilders.shift();
     const interfacePath = interfacePathBuilder(name);
-    try {
-      esModule = requireInterface(interfacePath);
-    }
-    catch (err) {
-      if (err.code !== 'MODULE_NOT_FOUND') {
-        throw err;
-      }
-    }
+    esModule = loadInterface(interfacePath);
```

Comment:
This is what I mean, only changing the file we need to.
I did this locally and it seems to work:
![image](https://user-images.githubusercontent.com/53777086/164590951-e4731d8e-44ff-4560-8706-ea79b75e2091.png)

It will make it easier to do future TGU's, especially since we barely know TGUI

```suggestion
    try {
      esModule = loadInterface(interfacePath);
    }
    catch (err) {
      if (err.code !== 'MODULE_NOT_FOUND') {
        throw err;
      }
    }
  }
```

Reply:
I see what you mean now. I can do that sure

Reply:
Sorted